apistar/apischema.py

```diff
@@ -14,16 +14,20 @@
 from apistar.routing import (
     Route, RoutesConfig, primitive_types, schema_types, walk
 )
+from apistar.settings import Settings
 from apistar.templating import Templates
 
 
 class APISchema(Document):
     @classmethod
-    def build(cls, app: App, base_url: http.URL=None):
+    def build(cls, app: App, settings: Settings=Settings({}), base_url: http.URL=None):
```

Comment:
In preference to this I'd suggest default to `None` instead, and do a...

    if settings is None:
        settings = Settings({})